practice-one/src/styles/sections/introducing.css

```diff
@@ -0,0 +1,131 @@
+.section-introducing {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 128px;
+}
+
+.summary {
+  margin-bottom: 81px;
+  width: 620px;
+  text-align: center;
+}
+
+.summary-heading {
+  margin-bottom: 24px;
+  font-size: var(--font-size-2xl);
+  line-height: 50px;
+}
+
+.summary-content {
+  font-size: var(--font-size-medium);
+  line-height: 26px;
+}
+
+.feature {
+  display: flex;
+  flex-direction: column;
+  width: 290px;
+}
+
+.feature-order {
+  position: relative;
+  width: 63px;
+  height: 63px;
+  background-image: url("../../assets/images/order-icon.png");
+
+  & .order-number {
+    font-size: var(--font-size-xl);
+    line-height: 36px;
+    font-weight: 800;
+    color: var(--color-text-tertiary);
+  }
+}
+
+.feature-heading {
+  font-size: var(--font-size-medium);
+  line-height: 50px;
+}
+
+.feature-content {
+  font-size: var(--font-size-small);
+  line-height: 26px;
+}
+
+.order-number {
+  position: absolute;
```

Comment:
Don't use `absolute` here. You can use display flex for the `feature-order` to center numerical order